polyfills/Symbol/tests.js

```diff
@@ -30,6 +30,7 @@ var arePropertyDescriptorsSupported = function () {
 	}
 };
 var supportsDescriptors = Object.defineProperty && arePropertyDescriptorsSupported();
+var strictModeSupported = (function(){ return this; }).call(null) === null;
```

Comment:
I don't know enough about the polyfill pipeline, but is this file implicitly executed in strict mode?  (After all,  polyfills/Symbol/polyfill.js clearly is not.)

This would be less ambigous as a test reader using something like this:
```suggestion
var strictModeSupported = (function(){ 'use strict'; return this; }).call(null) === null;
```

Reply:
Good idea! I've added this in https://github.com/Financial-Times/polyfill-library/pull/193/commits/7fdae22972c88346bdd75b9c76b03389d68efa2f